thetis/equation.py

```diff
@@ -75,13 +82,21 @@ def __init__(self, function_space):
 
     def mass_term(self, solution):
         """
-        Default mass matrix term for the used solution function space.
+        Returns default mass matrix term for the solution function space.
 
-        Can be overloaded in derived classes if needed.
+        :returns: UFL form of the mass term
         """
         return inner(solution, self.test) * dx
 
     def add_term(self, term, label):
+        """
+        Adds a term in the equation
+
+        :arg term: :class:`.Term` object to add_term
+        :arg string label: Assign a label to the term. Generally one of
+            'implicit', 'explicit', 'source' indicating how this term should be
+            treated in the time integrator.
```

Comment:
Could link to `SUPPORTED_LABELS`: "Valid labels are given by :attr:`SUPPORTED_LABELS`".